src/node_crypto.h

```diff
@@ -596,18 +596,21 @@ class Hash : public BaseObject {
   Hash(Environment* env, v8::Local<v8::Object> wrap)
       : BaseObject(env, wrap),
         mdctx_(nullptr),
-        md_len_(0) {
```

Comment:
Did you intend to remove this line? Now md_len_ will have an undefined value, probably non-zero.

Reply:
It shouldn't matter, it will be set in the JS constructor before any other calls are made.